test/integration/smoke/test_iso.py

```diff
@@ -600,6 +603,6 @@ def test_07_list_default_iso(self):
         # list ISO should list default ISOS (VM and xen tools)
 
         # ListIsos to list default ISOS (VM and xen tools)
-        self.get_iso_details("xs-tools.iso")
         self.get_iso_details("vmware-tools.iso")
+        self.get_iso_details("xs-tools.iso")
```

Comment:
This is a good use case for using map instead of calling the get_iso_details twice. Otherwise LGTM !!